Assets/Scripts/Network/ServerList.cs

```diff
@@ -19,10 +19,10 @@ public class ServerList
 {
     List<ServerRecord> Servers;
 
-    public static ServerRecord Local = new ServerRecord("Local", "ws://localhost:2567");
-    public static ServerRecord Custom = new ServerRecord("Custom", "ws://192.168.1.86:2567");
-    public static ServerRecord Dev = new ServerRecord("Dev", "wss://ceasar-serve-update-col-4va3ac.herokuapp.com/");
-    public static ServerRecord Web = new ServerRecord("Web", "wss://ceasar-server-staging.herokuapp.com/");
+    public static ServerRecord Local = new ServerRecord("Local", "ws://localhost:3000");
+    public static ServerRecord Custom = new ServerRecord("Custom", "ws://192.168.1.86:3000");
+    public static ServerRecord Dev = new ServerRecord("Dev", "wss://apps.concord.org/ceasar-server/");
```

Comment:
I assume the same url is ok for dev and production.  If not there is a QA app stack that would let you deploy to `apps.concordqa.org`.